internal/controllers/gateway/route_utils_test.go

```diff
@@ -1912,7 +1913,7 @@ func TestIsRouteAcceptedByListener(t *testing.T) {
 			expectedValue: true,
 		},
 		{
-			name: "not accepted, listener not ready",
+			name: "accepted, listener not ready",
```

Comment:
There's no `Accepted` condition in this test case, is that expected?

Reply:
Ah, the `accepted` in the test case name is about the expected test result. Nevermind. Maybe I'd only suggest rephrasing to `route accepted when listener not programmed`.

Reply:
Updated.